### DISCUSSION TITLE
Is there a way to configure a different port for postgresql in Grafana?

### DISCUSSION THREAD

**Discussion by marvinlei** [teslamate-org/teslamate#4102] [Getting started]
I'm trying to install Teslamate on my Synology server but port 5432 is already taken. So I started postgresl on a different port. Now Grafana won't show me anything because it's looking for the database on port 5432.

I tried passing DATABASE_PORT <my port> as an environmentvariable but to no avail.

Thanks for your help!

**Comment by boba51** [comment]
Also require help with this, frustrating issue, tried many things including uninstalling and reinstalling Container Manager.
On Synology DSM version 7.2.64561
Any help to keep my sanity would be appreciated.
